ProcessLib/GroundwaterFlow/GroundwaterFlowFEM.h

```diff
@@ -100,11 +100,12 @@ class LocalAssemblerData
                                  sm.detJ * wp.getWeight();
 
             // Darcy velocity only computed for output.
-            auto const darcy_velocity = -(k * sm.dNdx *
-                Eigen::Map<const NodalVectorType>(local_x.data(), ShapeFunction::NPOINTS)
-                ).eval();
+            NodalVectorType const darcy_velocity =
```

Comment:
is `NodalVectorType` really correct here? `darcy_velocity` should have a length of spatial dimension.